app/project.js

```diff
@@ -136,6 +136,13 @@ export default class Project extends Logger {
         const dist = item.path.replace(/\bapp\b/, 'dist')
 
         if (item.processor === 'template' && locale) {
+          const parts = dist.split(path.sep)
+          const locales_index = parts.indexOf('locales')
+          const folder = parts[locales_index + 1]
```

Comment:
Same thing here. This won't work because it won't always have a locale folder. You have to check to see if the index is `> -1`

Reply:
you can actually just run `path.normalize(dist.replace(/\blocales\b/, ''))` and it does the same thing